README.rst

```diff
@@ -83,6 +92,12 @@ Running The Tests
 Change Log
 ----------
 
+1.7
+^^^
+
+* Added marionette functionality
```

Comment:
I think https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette might be a better place to link to.